prediction/src/algorithms/identify/trained_model.py

```diff
@@ -34,8 +48,111 @@ def predict(dicom_path):
              'z': int,
              'p_nodule': float}
     """
-    load_dicom(dicom_path)
-    return [{'x': 0,
-             'y': 0,
-             'z': 0,
-             'p_nodule': 0.5}]
+    if dicom_path[-1] != '/':
+        dicom_path += '/'
+    dicom_files = glob.glob(dicom_path + "*.dcm")
+    if len(dicom_files) == 0:
```

Comment:
if `dicom_files` is a plain iterable, `if not dicom_files:` works (and is more pythonic as well :))...